source/appModules/microsoftedge.py

```diff
@@ -0,0 +1,17 @@
+# MicrosoftEdge.py
+#A part of NonVisual Desktop Access (NVDA)
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+#Copyright (C) 2018 NV Access Limited, Joseph Lee
+
+"""appModule for Microsoft Edge main process"""
+
+import appModuleHandler
+import ui
+
+class AppModule(appModuleHandler.AppModule):
+
+	def event_UIA_notification(self, obj, nextHandler, displayString=None, **kwargs):
+		# #8423: even though content process is focused, notifications are fired by main Edge process.
+		# The base object will simply ignore this, so notifications must be announced here and no more.
```

Comment:
If I understand this correctly, these notifications will be announced regardless of what application has the foreground. It is imaginable that notifications are fired even though either this or a child process doesn't have focus. Could you somehow make sure that notifications are only announced within the context of Edge?

Furthermore, how to make sure that this appModule loads? Does it ever receive focus and if so, when?